docs/pages_en/whats_new_in_v1_2/how_to_migrate_from_v1_1_to_v1_2.md

```diff
@@ -33,6 +33,10 @@ Commands changes (for reference):
  - `werf converge` will build and publish needed images by itself, if not built already.
  - You may use `werf build` command in `prebuild` CI/CD pipeline stage instead of `werf build-and-publish` command.
 
+**Please note!** The `werf converge` command stores both the collected images and their stages in the container registry. To specify the storage to be used for storing images and stages, you now need to specify one parameter `--repo` instead of the two previously used in v1.1 `--stages-storage` and `--images-repo`. You can read more about this parameter [in changelog](https://werf.io/documentation/v1.2/whats_new_in_v1_2/changelog.html#always-store-image-stages-in-the-container-registry).
+
+Similarly with the environment variables used earlier: instead of `WERF_STAGES_STORAGE` and `WEB_IMAGES_REPO`, one variable `WERF_REPO` is now used.
```

Comment:
```suggestion
Similarly with the environment variables used earlier: instead of `WERF_STAGES_STORAGE` and `WERF_IMAGES_REPO`, one variable `WERF_REPO` is now used.
```